terraform/services/bucket-access-logs/terraform.tf

```diff
@@ -0,0 +1,18 @@
+provider "aws" {
+  default_tags {
+    tags = {
+      application = var.app
+      business    = "oeda"
+      code        = "https://github.com/CMSgov/ab2d-bcda-dpc-platform/tree/main/terraform/services/bucket-access-logs"
+      component   = "bucket-access-logs"
+      environment = var.env
+      terraform   = true
```

Comment:
The app and env variables are no longer needed here since the bucket is at the account level. The variables file can then also be removed, and no need to set TF_VAR_app and TF_VAR_env in the workflows.